plugins/base/src/main/kotlin/renderers/html/shouldRenderSourceSetBubbles.kt

```diff
@@ -5,7 +5,7 @@ import org.jetbrains.dokka.pages.ContentPage
 import org.jetbrains.dokka.pages.RootPageNode
 
 internal fun shouldRenderSourceSetBubbles(page: RootPageNode): Boolean {
-    return page.withDescendants()
+    return shouldRenderSingleSourceSetBubble() || page.withDescendants()
```

Comment:
It needs to check that we have a single source set (count() == 1). 
Some pages (e.g. a multimodule page) do not have source sets.